examples/rspack/rspack.config.js

```diff
@@ -1,40 +1,53 @@
+const rspack = require("@rspack/core");
+
 /**
  * @type {import('@rspack/cli').Configuration}
  */
 module.exports = {
-    context: __dirname,
-    entry: {
-        main: "./src/main.tsx"
-    },
-    builtins: {
-        html: [
-            {
-                template: "./index.html"
-            }
-        ]
-    },
-    module: {
-        rules: [
-            {
-                test: /\.svg$/,
-                type: "asset"
-            },
-            {
-                test: /\.tsx$/,
+	context: __dirname,
+	entry: {
+		main: "./src/main.tsx"
+	},
+	resolve: {
+		extensions: [".js", ".ts", ".tsx", ".jsx"]
+	},
+	module: {
+		rules: [
+			{
+				test: /\.svg$/,
+				type: "asset"
+			},
+			{
+				test: /\.(jsx?|tsx?)$/,
                 exclude: /node_modules/,
-                use: {
-                    loader: 'babel-loader',
-                    options: {
-                        presets: [
-                            "@babel/preset-typescript",
-                            "@babel/preset-react"
-                        ],
-                        plugins: [
-                            "macros"
-                        ]
-                    }
-                }
-            }
-        ]
-    }
+				use: [
+					{
+						loader: "builtin:swc-loader",
+						options: {
+							sourceMap: true,
+							jsc: {
+								parser: {
+									syntax: "typescript",
```

Comment:
there is a problem with formatting. We use 2 spaces, here is used 4 spaces. Also part with "builtin:swc-loader" use even more spaces 

<img width="797" alt="image" src="https://github.com/lingui/js-lingui/assets/1586852/326c1681-95eb-4d08-a6d7-327a91e3084b">

Reply:
Thanks. I have addressed the changes you requested. PTAL.